File: app/src/main/java/com/lubenard/oring_reminder/pages/about/AboutFragment.java
package com.lubenard.oring_reminder.pages.about;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import com.lubenard.oring_reminder.BuildConfig;
import com.lubenard.oring_reminder.R;

public class AboutFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        return inflater.inflate(R.layout.about_fragment, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        ((AppCompatActivity)requireActivity()).getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        requireActivity().setTitle(R.string.about_additional_software_licenses);

        ((TextView)view.findViewById(R.id.app_version)).setText(BuildConfig.VERSION_NAME);
    }
}

File: app/src/main/java/com/lubenard/oring_reminder/pages/about/AboutFragment.kt
package com.lubenard.oring_reminder.pages.about

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.lubenard.oring_reminder.BuildConfig
import com.lubenard.oring_reminder.R

class AboutFragment: Fragment() {
    
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        super.onCreate(savedInstanceState)
        return inflater.inflate(R.layout.about_fragment, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        (requireActivity() as AppCompatActivity).supportActionBar?.setDisplayHomeAsUpEnabled(true)

        requireActivity().setTitle(R.string.about_additional_software_licenses)

        (view.findViewById<TextView>(R.id.app_version)!!).text = BuildConfig.VERSION_NAME
    }
}
